packages/osd-stylelint-plugin-stylelint/src/rules/no_custom_colors/index.ts

```diff
@@ -31,12 +32,12 @@ const messages = ruleMessages(ruleName, {
   expected: (message) => `${message}`,
 });
 
-const ruleFunction = (
+const ruleFunction: stylelint.Rule = (
   primaryOption: Record<string, any>,
   secondaryOptionObject: Record<string, any>,
   context
 ) => {
-  return (postcssRoot: any, postcssResult: any) => {
+  return (postcssRoot, postcssResult) => {
```

Comment:
Technical note - these types are inferred because we added the `stylelint.Rule` type